docs/ui/interactions/fragments/web/chatbot.md

```diff
@@ -0,0 +1,11 @@
+A simple way to add a conversational UI into your app is to use our ChatBot Component.
+
+ChatBot automatically renders a complete chat messaging interface that can be used out-of-the-box, or it can be customized using theming support.
+
+<inline-fragment src=~/ui/interactions/fragments/web/installation.md></inline-fragment>
```

Comment:
Let's add a callout section here describing the component's preview state.

Reply:
Done